docs/source/daml/reference/interfaces.rst

```diff
@@ -41,9 +44,10 @@ Interface Methods
    :end-before: -- INTERFACE_METHODS_END
 
 - An interface may define any number of methods.
-- Methods are in scope as functions at the top level, in the ensure clause, and
-  in interface choices. These functions always take an unstated first argument
-  corresponding to a contract that implements the interface:
+- Methods are in scope as functions at the top level and in interface choices.
```

Comment:
this sounds a bit confusing, how about something like "methods define top-level definitions". The scoping rules are the same as for any other top-level def.

Reply:
Thank you, I'll rephrase